stdlib/messaging/nats/src/main/ballerina/src/nats/commons.bal

```diff
@@ -20,7 +20,9 @@ public type Content byte[] | boolean | string | int | float | decimal | xml | js
 
 function convertData(Content data) returns string | byte[] | error {
     string | byte[] | error converted;
-    if (data is boolean) {
+    if (data is byte[]) {
```

Comment:
Can we check with the std lib team if this order change is OK? Since this used to be the last case, just to make sure we aren't making the least likely usage the first check.

Reply:
@daneshk @aashikam FYA please.

Reply:
Order doesn't matter. should be fine